frontend/javascripts/oxalis/model/sagas/volumetracing_saga.tsx

```diff
@@ -335,6 +344,18 @@ export function* editVolumeLayerAsync(): Saga<any> {
     );
 
     yield* put(finishAnnotationStrokeAction(volumeTracing.tracingId));
+
+    if (!wroteVoxelsBox.value) {
+      const overwriteMode = yield* select((state) => state.userConfiguration.overwriteMode);
+      if (overwriteMode === OverwriteModeEnum.OVERWRITE_EMPTY)
+        yield* call(
+          [Toast, Toast.warning],
+          "No voxels were changed. You might want to change the overwrite-mode to 'Overwrite All'. Otherwise, only empty voxels will be changed.",
```

Comment:
```suggestion
          "No voxels were changed. You might want to change the overwrite-mode to “Overwrite All” in the toolbar. Otherwise, only empty voxels will be changed.",
```

Maybe this will help first-time users find the button